nextflow_schema.json

```diff
@@ -516,6 +516,29 @@
                     "fa_icon": "fas fa-file",
                     "description": "The desired output format.",
                     "enum": ["tsv", "csv", "arrow", "parquet", "biom"]
+                },
+                "taxpasta_taxonomy_dir": {
+                    "type": "boolean",
+                    "description": "The path to a directory containing taxdump files.",
+                    "help_text": "At least nodes.dmp and names.dmp are required. A merged.dmp file is optional."
+                },
+                "taxpasta_add_name": {
+                    "type": "boolean",
+                    "description": "Add the taxon name to the output."
+                },
+                "taxpasta_add_rank": {
+                    "type": "boolean",
+                    "description": "Add the taxon rank to the output."
+                },
+                "taxpasta_add_lineage": {
+                    "type": "boolean",
+                    "description": "Add the taxon's entire lineage to the output.",
+                    "help_text": "These are taxon names separated by semi-colons.   "
```

Comment:
I would like to see more long form descriptions of what actually the flags do in the `help_text`, you're also missing for all of the taxpata parametes the 

```
> Modifies tool parameter(s)
     - taxpasta: `--<xyz>
```

Things (also for taxonomy above, just notice dthat sorry)

Reply:
Do you mean that I do not have a parameter for different taxpasta tool? I do not understand what I am missing, I have adressed the different parameters or do you mean something else?

Reply:
I will follow some of the current example in the `nextflow_schema.json`